src/index.js

```diff
@@ -9,8 +9,7 @@ app.set("views", "./views");
 app.set("view engine", "pug");
 app.use(express.static("public"))
 
-app.locals.fpjsRollbarClientId = process.env["PAYWALL_FPJS_ROLLBAR_CLIENT_ID"] || '';
-app.locals.fpjsRollbarToken = process.env["PAYWALL_FPJS_ROLLBAR_TOKEN"] || '';
+app.locals.fpjsCdnUrl = process.env["PAYWALL_FPJS_CDN_URL"] || 'https://fpcdn.io/';
```

Comment:
Please add an environment variable:

- `PAYWALL_FPJS_CDN_URL` = `https://c.fingerprintjs.com/` if the JS agent token belongs to the production environment
- `PAYWALL_FPJS_CDN_URL` = `https://procdn.fpjs.sh/` if the JS agent token belongs to the RC environment

Reply:
@Lisenish could you add them please?

Reply:
We don't have staging environment for paywall demo, added on prod

Reply:
@Lisenish Looks like the variable isn't added because the demo sends requests to https://fpcdn.io instead of https://c.fingerprintjs.com

Reply:
It's ok, my bad